lib/creator/yeoman/utils/entry.js

```diff
@@ -41,7 +41,7 @@ module.exports = (self, answer) => {
 			return forEachPromise(entryIdentifiers, (entryProp) => self.prompt([
 				InputValidate(
 					`${entryProp}`,
-					`What is the location of '${entryProp}'? [example: '../dist/app.js']`,
+					`What is the location of '${entryProp}'? [example: './app']`,
```

Comment:
FP = file path?

`./app` gets converted to './app.js'. Please run the generator yourself to see the output.